tests/cephfs/cephfs_utils.py

```diff
@@ -2583,3 +2583,18 @@ def get_files_and_checksum(self, client, directory):
             md5sum = out.read().decode().strip().split()
             file_dict[file] = md5sum[0]
         return file_dict
+
+    def fs_client_authorize(
```

Comment:
Hi  Yogesh,
Can you add documentation to this function.
So that it gets reflected in the sphinx documentation.